assignment-2-ruby-ac/annabel-assignment2.rb

```diff
@@ -0,0 +1,181 @@
+#Question 1: Customize the room_cleaner 
+
+room = ["kitchen", "bathroom", 'livingroom']
+
+def mopped(room)
+    puts "The #{room} has been mopped!"
+end
+
+def wash_dishes
+    puts "The dishes have been washed"
+end
+
+def wiped(room)
+    puts "The #{room} has been wiped down!"
+end
+
+def toilet_bowl
+    puts "The toilet has been cleaned"
+end
+
+def vaccumed(room)
+    puts "The #{room} has been vaccumed!"
+end
+
+def clean_house(room)
+    if room == "kitchen"
+        mopped(room)
+        wash_dishes
+    elsif room == "bathroom"
+        wiped(room)
+        toilet_bowl
+    elsif room == "livingroom"
+        vaccumed(room)
+    else
+        puts "The #{room} does not need a specific cleaning process"
+    end
+
+    mop_floor if room == "kitchen" || room == "bathroom"
+
+# I am asking what their cleaning list will be
+
+def create_cleaning_list
+    print "Please name your cleaning list for the cleaning bot: "
+    name = gets.chomp
+    return name
+end
+
+def add_rooms
+    print "What rooms would you like the bot to clean? (Kitchen, bathroom, livingroom)"
+    rooms = gets.chomp
```

Comment:
We should create this variable out of the function so that we can share the value across functions